README.md

```diff
@@ -35,3 +35,55 @@ But be aware: Discovery suffers revisions much more frequently, than this repo.
 Why a separate repo?
 At the inception of the Discovery project, the files inside this repo were buried within the Discovery repo. Neil split them out, 
 to facilitate RPM-based distribution.  And to prevent Sam from accidentally updating the schema, without telling me.
+
+# Docker image for UAL Discovery/NEOS Discovery Solr
+
+
+## What is this?
+
+This image can be used as a Solr datastore for [UAL Discovery](https://github.com/ualbertalib/discovery) and [NEOS Discovery](https://github.com/ualbertalib/neosdiscovery). This image is built on the [Solr 6.6 image](https://hub.docker.com/_/solr/) and adds the configuration files required to create collections. This is necessary because the configuration lives in a seperate repository to the application.
+
+## Requirements
+
+
+Ensure you have [Docker (1.13.0+)](https://docs.docker.com/engine/installation/) and 
+[Docker Compose (1.10.0+)](https://docs.docker.com/compose/install/) installed
+
+
+## In this Docker Container
+
+In the Docker Container are the minimum dependencies to run Solr with this projects schema and configurations available in the /myconfig directory.
+
+
+## Usage
+
+Expected usage is with Docker Compose `docker-compose up -d` which can use the entrypoints

```

Comment:
For instance, I'm currently able to use a docker-compose.yml as datastores for the discovery application.
```
version: '3'

volumes:
  solr:
    driver: local

services:
  solr:
    image: pgwillia/blacklight_solr_conf:test
    ports:
       - "8983:8983"
    volumes:
      - solr:/opt/solr/server/solr/mycores
    entrypoint:
      - solr-precreate
      - discovery-test
      - /myconfig
```